AnswerKingAPI.yml

```diff
@@ -2,15 +2,230 @@ openapi: 3.0.1
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+    put:
+      tags:
+        - Tags
+      summary: Modify Tag.
+      operationId:  modifyTag
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Tag.Request'
+      responses:
+        '200':
+          description:  OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Tag.Response'
+        '400':
+          description: Invalid parameters are provided.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+        '404':
+          description: Tag with the given id does not exist.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+    delete:
+      tags:
+        - Tags
+      summary:  Delete Tag.
+      operationId:  deleteTag
+      responses:
+        '204':
+          description:  No Content
+        '400':
+          description: Invalid parameters are provided.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+        '404':
+          description: Tag with the given id does not exist.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+        '410':
+          description: Tag with the given id is already retired.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+
+  /api/tags/{id}/products:
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+    get:
+      tags:
+        - Tags
+      summary:  List  Products For Tag.
+      operationId:  viewProductsForTag
+      responses:
+        '200':
+          description:  OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Inventory.Product.Response'
+        '400':
+          description: Invalid parameters are provided.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+        '404':
+          description: Tag with the given id does not exist.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+    put:
+      tags:
+        - Tags
+      summary:  Assign tag to products.
+      operationId:  assignTagToProducts
+      requestBody:
+        description: ''
```

Comment:
Does this one need an empty description? could probably remove if there's no description for it

Reply:
I deleted it